csfieldguide/curriculum_guides/content/en/apcsp/sections/creativity.md

```diff
@@ -1,24 +1,29 @@
 # Creativity
 
 ## Overview
+
 - EU 1.1 Creative development can be an essential process for creating computational artifacts.
 - EU 1.2 Computing enables people to use creative development processes to create computational artifacts for creative expression or to solve a problem.
 - EU 1.3 Computing can extend traditional forms of human expression and experience.
 
 ## Reading from the Computer Science Field Guide
+
 Start by reading through:
 
-- Human Computer Interaction
- - [Interface Usability](chapters/human-computer-interaction.html#interface-usability)
+- [Human Computer Interaction - Interface Usability]('chapters:chapter_section' 'human-computer-interaction' 'interface-usability')
 
 ## Learning objectives
-The above chapter readings include specific knowledge for EK's marked in bold. Work to include unmarked learning objectives in the CS Field Guide is currently in progress.
+
+The above chapter readings include specific knowledge for EKs marked in bold.
+Work to include unmarked learning objectives in the CS Field Guide is currently in progress.
 
 ### LO 1.1.1 Apply a creative development process when creating computational artifacts.
+
 - EK 1.1.1A A creative process in the development of a computational artifact can include, but is not limited to, employing nontraditional, nonprescribed techniques; the use of novel combinations of artifacts, tools, and techniques; and the exploration of personal curiosities.
 - EK 1.1.1B Creating computational artifacts employs an iterative and often exploratory process to translate ideas into tangible form.
 
 ### LO 1.2.1 Create a computational artifact for creative expression.
+
 - EK 1.2.1A A computational artifact is something created by a human using a computer and can be, but is not limited to, a program, an image, audio, video, a presentation, or a Web page file.
```

Comment:
Should web page be lowercase?

Reply:
Commented below on this, in short: it's capitalised everywhere so you can make a call.

Reply:
It's meant to be one word